### ISSUE TITLE
Trying to make a new Pipeline, but the scheduler doesn't apply the denoising step

### ISSUE THREAD

**Issue by alexblattner** [huggingface/diffusers#3317]
Hello, I am creating a new pipeline for latent couple which is essentially a way to specify prompts on parts of the picture. Here's the code I'm trying to replicate:
https://colab.research.google.com/drive/1UdElpQfKFjY5luch9v_LlmSdH7AmeiDe?usp=sharing

I copied the latest StableDiffusionPipeline class and modified it. I added the file in here (as txt because github won't allow otherwise)
[Pipeline.txt](https://github.com/huggingface/diffusers/files/11377539/Pipeline.txt)

I ran this code to make it work:
```
import torch
from diffusers import StableDiffusionControlNetPipeline, ControlNetModel, UniPCMultistepScheduler,ModelMixin
from io import BytesIO
from transformers import AutoImageProcessor, UperNetForSemanticSegmentation
from PIL import Image
import torch.multiprocessing as mp
import random
from compel import Compel
import Loras
import numpy as np
from multilatents import LatentCouplesPipeline
pipe = LatentCouplesPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16,safety_checker=None,requires_safety_checker=False)
pipe = pipe.to("cuda")
compel_proc = Compel(tokenizer=pipe.tokenizer, text_encoder=pipe.text_encoder)
prompt=["park, masterpiece, high quality, 8k, (blurry)0.7", "jumping dog, masterpiece, high quality, 8k"]
# promptE=compel_proc(prompt)
generator = torch.manual_seed(1)
pos=["1:1-0:0","1:3-1:1"]
mix_val=[0.2,0.9]
print(pos)
image = pipe(
    prompt=prompt,
    pos=pos,
    mix_val=mix_val,
    num_inference_steps=20,
    generator=generator,
).images[0]
image.save("Lout.png")
```
it essentially got stuck at step 1 during the denoising. If this works, we could add latent couples as a community pipeline! I'd love to do one for controlnet and multicontrolnet with latent couples afterwards

**Comment by alexblattner** [comment]
@timegate @showxu
is this something you guys know about?

**Comment by alexblattner** [comment]
I ended up finishing it. Here's the pipeline my dudes
[Pipeline.txt](https://github.com/huggingface/diffusers/files/11384037/Pipeline.txt)

here's how to use it (doesn't work with embeddings yet):
```
pipe = LatentCouplesPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16,safety_checker=None,requires_safety_checker=False)
pipe = pipe.to("cuda")
prompt=["((ultra-detailed)), ((illustration)), 2girls", "((ultra-detailed)), ((illustration)), 2girls, black hair","((ultra-detailed)), ((illustration)), 2girls, blonde hair"]
negative_prompt=["(low quality, worst quality:1.4)", "(low quality, worst quality:1.4)","(low quality, worst quality:1.4)"]
# promptE=compel_proc(prompt)
generator = torch.manual_seed(1)
pos=["1:1-0:0","1:2-0:0","1:2-0:1"]
mix_val=[0.1, 0.9, 0.9]
image = pipe(
    prompt=prompt,
    negative_prompt=negative_prompt,
    pos=pos,
    mix_val=mix_val,
    num_inference_steps=20,
    generator=generator,
).images[0]
image.save("Lout1.png")
```

**Comment by imiraoui** [comment]
Would have been nice to merge?

**Comment by kadirnar** [comment]
@sayakpaul , @patrickvonplaten

**Comment by sayakpaul** [comment]
https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines is where the source code of the core pipelines live. I recommend checking them out.

**Comment by kadirnar** [comment]
> https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines is where the source code of the core pipelines live. I recommend checking them out.

Yes I know that. But this is a new feature. We need to open a pull request. Do @alexblattner  want to do it? I can do it too. We should add this support.

**Comment by alexblattner** [comment]
@kadirnar I'll update my code for my version of diffusers and will help you then :)

**Comment by kadirnar** [comment]
> @kadirnar I'll update my code for my version of diffusers and will help you then :)

Great😍 I'm waiting for the updated code. It would be great to test with SDXL.

**Comment by alexblattner** [comment]
@kadirnar it's without lora switching yet, but it's as good as it can be in all other aspects: https://github.com/alexblattner/RubberDiffusers
